specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/OperationalInsights.json

```diff
@@ -0,0 +1,429 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/DataExport"
+            }
+          },
+          "404": {
+            "description": "Not found the specific data export."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DataExport"
+        ],
+        "x-ms-examples": {
+          "DataExportDelete": {
+            "$ref": "./examples/DataExportDelete.json"
+          }
+        },
+        "operationId": "DataExport_Delete",
+        "description": "Deletes the specified data export in a given workspace..",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "name": "dataExportName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The data export rule name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition."
+          },
+          "404": {
+            "description": "Not found the specific data export."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataExport": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DataExportProperties",
+          "description": "data export properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The top level data export resource container.",
+      "x-ms-azure-resource": true
+    },
+    "DataExportListResult": {
+      "description": "Result of the request to list data exports.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataExport"
+          },
+          "description": "List of data export instances within a workspace.."
+        }
+      }
+    },
+    "DataExportProperties": {
+      "properties": {
+        "dataExportId": {
```

Comment:
>dataExportId [](start = 9, length = 12)

If this is a fixed set, consider defining as enum and modelAsString=true for flexibility to add a new one without breaking clients

Reply:
It's not a fixed set. It a unique guid generator identifier